Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -84,6 +93,22 @@ - (void)track:(SEGTrackPayload *)payload
         return;
     }
     
+    NSDictionary *adobeVideoEvents = @{
+                                       @"Video Playback Started" : @"trackSessionStart",
```

Comment:
The values in this dictionary are unused.  I see that you call `hearbeat. trackSessionStart `, but nothing is actually using the string `@"trackSessionStart"`.

Reply:
good catch, will remove now